index.js

```diff
@@ -1 +1,63 @@
 // implement your API here
+// import express from 'express' //es2015 Module Import
+const express = require('express');
+
+const db = require('./data/db.js')
+
+const server = express();
+
+server.use(express.json()); //tells express how to write json
+// does not know how to do out of the box
+
+    server.get('/', (req, res) => {
+            res.send("It's alive!");
+        })
+
+    server.post('/api/users', (req, res) => {
+
+    const userInfo = req.body;
+    console.log('requst body', userInfo);
+    
+    db.insert(users)
+        .then( users => {
+            res.status(200).json(users)
+        })
+        .catch( error => {
+            res.status(500).json({error: error, message: "error with user"})
+        })
+    })
+
+    server.get('/api/users', (req, res) => {
+        db.find(users => {
+            res.status(200).json(users);
```

Comment:
right here you're using the `find` function on your database (db), but then you're actually giving the find function a callback function, which it doesn't take in. instead you'll wanna have ```db.find().then(users => etc)```, the `.then` allowing you to deal with whatever db.find() returned